lib/oban/job.ex

```diff
@@ -215,13 +215,13 @@ defmodule Oban.Job do
     |> Map.new()
   end
 
-  defp coerce_field(params, field) do
+  defp coerce_field(params, field, func) do
```

Comment:
For consistency with other anonymous function passing let's use `fun` instead of `func` here:

```suggestion
  defp coerce_field(params, field, fun) do
```